src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/Microsoft.Build.NuGetSdkResolver.csproj

```diff
@@ -4,8 +4,7 @@
 
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;net46</TargetFrameworks>
-    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
```

Comment:
are you removing support for `IsBuildOnlyXPLATProjects`? Should you take this into account in `common.props`?

Reply:
It hasn't been relevant in a while now. 
See https://github.com/NuGet/NuGet.Client/commit/7be2fe5b0d1fed0a67dc69a8082c14f9b616e9b0#diff-312424e272a66fd937741fda8e21aaff. 

I think it was a "race condition" between when this project was added and when those things were removed.